### ISSUE TITLE
How does it work with the image from URL?

### ISSUE THREAD

**Issue by Laurence-Remotely** [mattiasw/ExifReader#186]
### Description
How does it work with the image from URL?

**Additional details**

- ExifReader version: 4.3.1
- Web browser and version: Edge 99
- Node version: 16
- Can you reproduce the bug on the examples site? If so, using which implementation (global object, AMD, and/or ES module)? https://mattiasw.github.io/ExifReader/

**How to reproduce**
I understand I need to use ArraryBuffer to load the tags. Since the image I want to load is from the cloud, so I use this way to load image from the URL and convert it as an ArrayBuffer (in the browser):

```
const getImageFormUrl = async (url: string) => {
  const img = new Image();
  img.setAttribute('crossOrigin', 'anonymous');
  img.onload = function () {
    const canvas = document.createElement('canvas');
    canvas.width = img.width;
    canvas.height = img.height;

    canvas.toBlob(async (blob) => {
      if (blob) {
        const arrayBuffer = await blob.arrayBuffer();
        const tags = ExifReader.load(arrayBuffer);
        console.log(tags);
      }
      loadImage.value = false;
    });
  };
  img.src = url;
};
```
**What I expected would happen:**
I expect I can see Exif tags listed as example website. (I downloaded the image from the URL and uploaded it to the example website, and I could see the Exif tags correctly)

**What really happened:**
I can only see those tags when I use console.log to print out the tags
```
Bit Depth: {value: 8, description: '8'}
Color Type: {value: 6, description: 'RGB with Alpha'}
Compression: {value: 0, description: 'Deflate/Inflate'}
Filter: {value: 0, description: 'Adaptive'}
Image Height: {value: 480, description: '480px'}
Image Width: {value: 720, description: '720px'}
Interlace: {value: 0, description: 'Noninterlaced'}
```

**Comment by mattiasw** [comment]
Hi! When an image is converted to a canvas the browser throws away all the meta data. The little you can still see is the regular image details. What you can do instead, if your cloud server has the correct CORS headers turned on, is to pass in the URL directly like so:
```
const tags = await ExifReader.load(url);
console.log(tags);
```

**Comment by Laurence-Remotely** [comment]
Hi Mattias, 
Thanks for the reply!

I tried this way before I converted image to canvas. It didn't work because the function doesn't accept `String` type as parameter.
In exif-reader.d.ts:
`export function load(data: ArrayBuffer | SharedArrayBuffer | Buffer): Tags & XmpTags & IccTags;`
It only accept `...Buffer`. exif-reader.d.ts might need an update?

**Comment by mattiasw** [comment]
Yes, you're right, it needs an update. I didn't think about that since I don't use TypeScript with ExifReader myself. The file was created and has been maintained by other ExifReader users sending in pull requests. But I tried myself now. Do you think it looks correct? https://github.com/mattiasw/ExifReader/pull/187/files

**Comment by LaurenceHo** [comment]
> Yes, you're right, it needs an update. I didn't think about that since I don't use TypeScript with ExifReader myself. The file was created and has been maintained by other ExifReader users sending in pull requests. But I tried myself now. Do you think it looks correct? https://github.com/mattiasw/ExifReader/pull/187/files

(I am the same person as above by using another account. Actually it's my main account :) )
Yes, it looks correct to me.

**Comment by mattiasw** [comment]
Cool. It's now released as version 4.4.0. Thanks for the help!
